spec/rest/tls.test.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
+
+define(['ably', 'shared_helper', 'async'], function(Ably, helper, async) {
+	var currentTime, exports = {};
```

Comment:
I thought about this myself but for some reason it didn't work for me without time. Thinking about it now, maybe it's because on this branch I did not have your fix on issue 205. I will calibrate my clock and try again but today I'm facing a different error when running these tests.

    Ably: PaginatedResource.get(): Unexpected error getting resource: err = {"statusCode":401,"code":40160,"message":"action not permitted, app = g8g-4A","serverId":"frontend.e821.eu-central-1-A.i-586ba0e5"}

Investigating.